### PR TITLE
Update extensible-admission-controllers.md

### PR DESCRIPTION
**Pull request by ningning54321**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.11 Features: set Milestone to 1.11 and Base Branch to release-1.11
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ This page describes how to use Admission Webhooks and Initializers.
 
 Admission webhooks are HTTP callbacks that receive admission requests and do
 something with them. You can define two types of admission webhooks,
-[validating admission Webhook](/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-18-beta-in-19)
+[validating admission Webhook](/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-1-8-beta-in-1-9)
```
Comment (ajaeger) [inline content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md]: The link should be: docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook-beta-in-1-9

File: content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 2]
```diff
@@ -33,9 +33,9 @@ This page describes how to use Admission Webhooks and Initializers.
...
 and
-[mutating admission webhook](/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-19).
+[mutating admission webhook](/docs/admin/admission-controllers/#mutatingadmissionwebhook-beta-in-1-9).
```
Comment (ajaeger) [inline content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md]: Update to new link location as well - /docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook-beta-in-1-9

File: content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md (hunk 1 above)
Comment (ningning54321) [reply content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md]: I did some update based on your comments.
While this updated url seems not working properly with Chrome, the achor doesn't works.
Same url works fine in safari.

My original url: /docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-1-8-beta-in-1-9
works for both browsers.

Comment (ajaeger) [reply content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md]: https://k8s.io//docs/admin/admission-controllers/ redirects to 
https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/

So, let's use the new URL.

**Pull request merged**